packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/xdd/DocumentResource.java

```diff
@@ -0,0 +1,79 @@
+package software.uncharted.terarium.hmiserver.resources.xdd;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import software.uncharted.terarium.hmiserver.models.xdd.Document;
+import software.uncharted.terarium.hmiserver.models.xdd.XDDArticlesResponseOK;
+import software.uncharted.terarium.hmiserver.models.xdd.XDDResponse;
+import software.uncharted.terarium.hmiserver.proxies.xdd.DocumentProxy;
+import software.uncharted.terarium.hmiserver.services.xdd.DocumentService;
+
+@Path("/api/xdd/documents")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Tag(name = "XDD Documents REST Endpoint")
+public class DocumentResource {
+
+		@Inject
+		DocumentService documentService;
+
+		@RestClient
+		DocumentProxy proxy;
+
+		// NOTE: the query parameters match the proxy version and the type XDDSearchPayload
+		@GET
+		@Produces(MediaType.APPLICATION_JSON)
+		@Tag(name = "Get all xdd documents via proxy")
+		public Response getDocuments(
+			@QueryParam("doi") String doi,
+			@QueryParam("title") String title,
+			@QueryParam("term") String term
+		) {
+			List<Document> documents = new ArrayList<>();
```

Comment:
remove all this and just return the response from the proxy
`return proxy.getDocuments(doi, title, term);`

Reply:
done